app/standalone.go

```diff
@@ -231,7 +231,7 @@ func doStandaloneInstallStates(art io.ReadCloser, key []byte,
 	}
 
 	if rollbackSupport {
-		fmt.Println("Use -commit to update, or -rollback to roll back the update.")
+		fmt.Println("Use commit to update, or rollback to roll back the update.")
```

Comment:
Suggestion:
Maybe we should single-quote the command line arguments now, that nothing is making them stand out from the rest of the text?

Reply:
I thought about the same issue (command not standing out) but I wasn't sure about the solution. But now that you commented on it, makes sense to add :+1: 

I will not re-run the full pipeline, though, as this exact log line seems not to be tested for.